tests/unit/test_ticket.py

```diff
@@ -1,45 +1,26 @@
-from models.tickets import TicketModel, TicketStatus
-from models.notes import NotesModel
-
-def test_ticket():
-    """The properties must match the ticket model's inputs."""
-    testIssue = 'Property Damage'
-    testSender = 'user1 tester'
-    testTenant = 'Renty McRenter'
-    testStatus = TicketStatus.New
-    testUrgency = 'low'
-    testAssignedUser = 4
-
-    test_ticket = TicketModel(
-        issue=testIssue,
-        sender=testSender,
-        tenant=testTenant,
-        status=testStatus,
-        urgency=testUrgency,
-        assignedUser=testAssignedUser,
-    )
-
-    assert test_ticket.issue == testIssue
-    assert test_ticket.sender == testSender
-    assert test_ticket.tenant == testTenant
-    assert test_ticket.status == testStatus
-    assert test_ticket.urgency == testUrgency
-    assert test_ticket.assignedUser == testAssignedUser
-
-
-def test_notes():
-    """The properties must match the notes model's inputs."""
-
-    testTicketId = 1
-    testText = 'Tenant has 40 cats'
-    testUser = 'user1 tester'
-
-    test_note = NotesModel(
-        ticketid=testTicketId,
-        text=testText,
-        user=testUser,
-    )
-
-    assert test_note.ticketid == testTicketId
-    assert test_note.text == testText
-    assert test_note.user == testUser
+from tests.unit.base_interface_test import BaseInterfaceTest
+from models.tickets import TicketModel
+from schemas.ticket import TicketSchema
+
+class TestBaseTicketModel(BaseInterfaceTest):
+    def setup(self):
+        self.object = TicketModel()
+        self.custom_404_msg = 'Ticket not found'
+        self.schema = TicketSchema
+
+    def test_validate_tenant(self):
+        invalid_tenant = {
+            'tenantID': 666
+        }
+
+        invalid_tenant_validation_error = {'tenantID': ['666 is not a valid tenant ID']}
+        assert invalid_tenant_validation_error == TicketSchema().validate(invalid_tenant)
+
+    def test_validate_assigned_user_and_sender(self):
+        invalid_assigned_user_and_sender = {
+            'assignedUserID': 777,
+            'senderID': 888
+        }
+
+        invalid_assigned_user_validation_error = {'assignedUserID': ['777 is not a valid user ID'], 'senderID': ['888 is not a valid user ID']}
+        assert invalid_assigned_user_validation_error == TicketSchema().validate(invalid_assigned_user_and_sender)
```

Comment:
Very nice. I think these tests are better suited in the `tests/schemas/test_ticket_schema.py` since it is testing what the schema does (input validation), and not testing what the model does (ticket creation/querying). However, let's get this merged and we can just make a mental note to move these when appropriate.